config/coreboot-nitropad-ns50.config

```diff
@@ -114,23 +119,23 @@ CONFIG_DIMM_SPD_SIZE=512
 CONFIG_FMDFILE=""
 # CONFIG_NO_POST is not set
 CONFIG_MAINBOARD_VENDOR="Notebook"
-CONFIG_CBFS_SIZE=0x1000000
```

Comment:
Definitely changing this was wrong as it makes the board build fail for ns50. Will revert that change later.

Reply:
@daringer: reverted changes under https://github.com/linuxboot/heads/pull/1640/commits/6582f49ddfa509ae640b5c8202590930aa1e79d5